django_select2/forms.py

```diff
@@ -311,8 +311,10 @@ def filter_queryset(self, term):
         """
         qs = self.get_queryset()
         search_fields = self.get_search_fields()
-        select = reduce(lambda x, y: x | Q(**{y: term}), search_fields,
-                        Q(**{search_fields.pop(): term}))
+        select = Q()
+        for t in term.split(' '):
```

Comment:
@codingjoe I think you better `replace` all occurrences of `\n` and `\t` by spaces then `strip` the term, then you can `split`, should be cleaner that way.